mne/preprocessing/tests/test_ica.py

```diff
@@ -763,14 +763,40 @@ def test_ica_twice(method):
 
 
 @requires_sklearn
-@pytest.mark.parametrize("method", ["fastica", "picard"])
+@pytest.mark.parametrize("method", ["fastica", "picard", "infomax"])
 def test_fit_params(method):
     """Test fit_params for ICA."""
     _skip_check_picard(method)
     fit_params = {}
     ICA(fit_params=fit_params, method=method)  # test no side effects
     assert_equal(fit_params, {})
 
+    # Test I/O roundtrip.
+    # Only picard and infomax support the "extended" keyword, so limit the
+    # tests to those.
+    if method in ['picard', 'infomax']:
+        tempdir = _TempDir()
```

Comment:
just use `tempdir = str(tmpdir)` after adding `tmpdir` fixture above. `_TempDir` is really for the most part a workaround for back when we used `nose`